.azure-pipelines/baseline_test/baseline.test.template.yml

```diff
@@ -86,7 +86,7 @@ jobs:
       - template: ../run-test-elastictest-template.yml
         parameters:
           TOPOLOGY: $(TOPOLOGY)
-          SCRIPTS: $(TEST_SCRIPTS)
+          SCRIPTS: $(SCRIPTS)
```

Comment:
What is the source of `$(SCRIPTS)`

Reply:
I realized it used to be $(SCRIPTS) before but I had accidentally changed it to $(TEST_SCRIPTS).  The source for it is https://github.com/sonic-net/sonic-mgmt/blob/master/.azure-pipelines/impacted_area_testing/calculate-instance-numbers.yml#L29